Document path options (#201)
Behat/docs: user_guide/configuration/printing_paths.rst

Printing paths
==============

> When showing test results or failures Behat will print the path to the related files. Usually it will
> simply print a path which is relative to the current directory but there are some configuration options
> that will let you modify this:

Reviewer: I think "By default" is a little clearer than "Usually" - if working in a shared project
then "usual" will depend on whether someone has already put these options in the
config file?

```suggestion
When showing test results or failures Behat will print the path to the related files. By default 
the path will be relative to the current directory. This behaviour can be modified through 
configuration or on the command line:
```

``Print Absolute Paths``: if this is set to true, Behat will print the full absolute path instead.

``Editor URL``: if this is set, Behat will surround each path with a link that, when clicked, will open

> the file in your IDE. This parameter is a template which can use some placeholders which will be
> replaced at run time by Behat. The available placeholders are

Reviewer: ```suggestion
the file in your IDE. This parameter is a template which can include placeholders that Behat will replace
at run time. The available placeholders are
```
